lib/sycamore/sycamore/tests/manual/test_fast_sycamore_import.py

```diff
@@ -0,0 +1,32 @@
+"""WARNING: this test needs to be run separately from other tests because if they
+   import sycamore and run it, then the checks for no direct depenencies will be
+   incorrect. Similarly, this test has to be run directly as python. pytest will
+   pre-import sycamore."""
+
+import inspect
+import sys
+
+
+def test_00_sycamore_not_yet_imported():
+    assert "sycamore" not in sys.modules, "Test run with other tests that import ray"
+
+
+def test_no_direct_ray():
+    assert "ray" not in sys.modules, "Test run with other tests that import ray"
+
+    assert "ray" not in sys.modules
```

Comment:
why twice?

Reply:
Because the line in between that did import sycamore got lost somehow. The test was uninteresting.

Reply:
The automatic linting stuff we had removed it.